script/api.py

```diff
@@ -103,12 +105,12 @@ def login():
 
     if session.get("user_id") is None:
         session["user_id"] = user_id
-        return jsonify({"Username": username})
+        return jsonify({"Username": username,"token":"test123"})
```

Comment:
Let's check strategies to generate a random token, one way is to use a base64 token or a uuid token. You may explore both.

Reply:
base64 is an encoding method. For that we still need to provide an input. I think the only way to go is to use uuid, but we will need to store that somewhere first so that we can check it later on

Reply:
True, we have to use uuid, but why do we need to store, we are sending this in API on the front end to be used and stored. Backend already has a session.